app/tests/profiles_tests/test_validators.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from django.core.exceptions import ValidationError
+
+from grandchallenge.profiles.validators import username_is_not_email
+
+
+def test_username_is_not_email():
+    username_is_not_email("test")
+
+
+def test_username_is_email_validator():
+    with pytest.raises(ValidationError):
+        username_is_not_email("test@gmail.com")
```

Comment:
See if you can re-write these tests as one [using parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html) (hint: you will need to use `null_context`, example should be in the codebase).

Reply:
rewritten!